unity/CAVE/Assets/Resources/CLI.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.IO;
+using System.Xml;
+using System.Text.RegularExpressions;
+using UnityEngine;
+using UnityEditor.SceneManagement;
+using UnityEditor.SceneTemplate;
+
+public class CLI : MonoBehaviour
+{
+    static void Main()
+    {
+        string xmlPath = null;
+
+        // Get command line arguments from Python
+        try {
+            string[] args = System.Environment.GetCommandLineArgs();
+            for(int i = 0; i < args.Length; i++)
+            {
+                if(args[i] == "--xmlPath") {
+                    xmlPath = args[i + 1];
+                    break;
+                }
+            }
+        } catch(Exception e) {
+            Debug.Log("Error initializing command line arguments");
+            Debug.Log(e);
+            throw e;
+        }
+
+        // Load the XML
+        try {
+            XmlDocument file = new XmlDocument();
+            file.Load(xmlPath);
+        } catch(FileNotFoundException e) {
+            Debug.LogError($"ERROR: File {xmlPath} not found");
+            Debug.LogException(e);
+            throw e;
+        } 
+
+        // Create the Unity scene
+        try{
+            NewScene(xmlPath);
+        } catch(Exception e) { 
+            Console.WriteLine($"Error creating scene for {xmlPath}");
+            Console.WriteLine(e);
+            throw e;
```

Comment:
Shouldn't these be `Debug.LogError` too?

Reply:
Oh good catch! I'll update that in #61